File: client/src/app/hrmanager/models/HRDashboard.ts
export interface HRDashboard {
    totalJobRequest: number;
    totalCandidates: number;
    totalJobConfirmed: number;
    totalInvitesCandidates: number;
    agencies: IAgency[];
    candidates: ICandidate[];
  }

export interface ICandidate {
    id: number;
    firstName: string;
    lastName: string;
    address1: string;
    address2: string;
    address3: string;
    address4: string;
    address5: string;
    contactNumber: string;
    email: string;
    accoutNumber: string;
    accoutName: string;
    sortCode: string;
    photoUrl: string;
    appUserId: string;
    grade: string;
    gradeId: number;
  }

export interface IAgency {
    id: number;
    name: string;
    contactName: string;
    address1: string;
    address2: string;
    address3: string;
    address4: string;
    address5: string;
    contactNumber: string;
    email: string;
    accoutNumber: string;
    accoutName: string;
    sortCode: string;
    logoUrl: string;
    appUserId: string;
    appUser?: any;
  }

File: Core/EntityHelpers/HRDashboard.cs
using Core.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace Core.EntityHelpers
{
    public class HRDashboard
    {
        public int TotalJobRequest { get; set; }
        public int TotalCandidates { get; set; }
        public int TotalJobConfirmed { get; set; }
        public int TotalInvitesCandidates { get; set; }
        public IReadOnlyList<Agency> Agencies { get; set; }
        public IReadOnlyList<Candidate> Candidates { get; set; }

    }
}
